experimental/overhead_matching/swag/data/vigor_dataset.py

```diff
@@ -4,10 +4,18 @@
 
 from pathlib import Path
 import pandas as pd
+import numpy as np
 from scipy.spatial import cKDTree
 from typing import NamedTuple
 
 
+class VigorDatasetConfig(NamedTuple):
+    panorama_neighbor_radius: float
+    satellite_patch_size: None | tuple[int, int]
+    panorama_size: None | tuple[int, int]
+    factor: None | float
```

Comment:
will this stick around in the dataset? If so may be helpful to explain
```suggestion
    factor: None | float  # if not None masks the dataset to factor% of the lat/long spanned by the original dataset. Should be [0, 1]
```